1_fetch/src/download_DO_data.R

```diff
@@ -0,0 +1,30 @@
+download_daily_mean_DO_data <- function(site_list,pcode,stat_cd,fileout){
+  
+  daily_DO_data_ls <- lapply(site_list,function(x){
+    readNWISdv(siteNumbers = x,parameterCd=pcode,statCd=stat_cd,startDate = "",endDate = "") %>%
+      renameNWISColumns()
```

Comment:
`readNWISdv()` comes from the dataRetrieval package and downloads the available daily data for a given site and parameter code(s). Would it help clarify the code chunk if I explicitly referenced dataRetrieval here, e.g. `dataRetrieval::readNWISdv(...)`? I'm not sure if USGS-R has style guidelines for this sort of thing.

Reply:
`renameNWISColumns` is also a helper function from dataRetrieval, so similarly, I could make that reference explicit if it would help with code readability.

Reply:
Ah. Okay. I don't think there is a USGS-R style guideline for this. I guess it gets back to a fundamental R thing that leaves ambiguity when it comes to the source of a given function. I'll defer to you (and @galengorski, if you have an opinion) since I am a less frequent R user, whether to specify the source (e.g., `dataRetrieval::renameNWISColumns`) or leave it as is since it seems like R folks wouldn't be phased by this.

Reply:
I'm hoping that adding better function documentation as discussed below will help address issues like this (so thanks for pointing it out!), however I've also decided to adopt source notation (e.g. `dataRetrieval::renameNWISColumns`) within functions to make external function definitions more clear.